src/intersection-observer-polyfill.js

```diff
@@ -286,6 +312,9 @@ export class IntersectionObserverPolyfill {
     for (let i = 0; i < this.observeEntries_.length; i++) {
       if (this.observeEntries_[i].element === element) {
         this.observeEntries_.splice(i, 1);
+        if (this.observeEntries_.length <= 0) {
+          this.disconnect();
```

Comment:
`disconnect` isn't entirely correct here, since it's API is defined by `Service`, not the InOb Polyfill.

Please add a `disconnectMutationObserver_` method.

Reply:
this.disconnect() call seems not needed here